core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksInodeStore.java

```diff
@@ -165,8 +165,18 @@ && new String(columns.get(2).getName()).equals(EDGES_COLUMN),
         PropertyKey.MASTER_METASTORE_ROCKS_EDGE_BLOCK_INDEX, mToClose)
         .ifPresent(cfg -> columns.get(1).getOptions().setTableFormatConfig(cfg));
 
-    mRocksStore = new RocksStore(ROCKS_STORE_NAME, dbPath, backupPath, opts, columns,
+    if (Configuration.getBoolean(PropertyKey.MASTER_PARALLEL_BACKUP_ROCKSDB)) {
+      String parallelBackupPath = PathUtils.concatPath(baseDir,
+          INODES_DB_NAME + "-parallel-backup");
+      Integer parallelBackupPoolSize = Configuration.getInt(
+          PropertyKey.MASTER_PARALLEL_BACKUP_ROCKSDB_THREAD_POOL_SIZE);
+
+      mRocksStore = new RocksStore(ROCKS_STORE_NAME, dbPath, backupPath, parallelBackupPath,
```

Comment:
Instead of adding the `parallelBackupPath` and `parallelBackupPoolSize`, you can resolve the property keys inside the util method so this argument list doesn't go longer and longer.

Reply:
parallelBackupPoolSize has been fixed, parallelBackupPath see the conversation below